common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -287,7 +288,7 @@ def _get_user(self):
         """
         Return the current runtime Django user.
         """
-        return get_user_model().objects.get(id=self.runtime.user_id)
+        return crum.get_current_user()
```

Comment:
I'm not sure if it is risky or safe to use crum.get_current_user() here. However, if I retrieve the user the previous way, it would not have the masquerade information on the user.

Reply:
I would have expected this to mess up tests that didn't set a current request. I think this is a fine approach - we should always have a user & request in the context of an xblock I think.

Feels like maybe the runtime should grow a `user` attribute instead of just a `user_id` attribute? But not your concern here. Your change also avoids a database hit, which is nice.